server/src/internalClusterTest/java/org/opensearch/index/IndexingPressureIT.java

```diff
@@ -74,11 +74,15 @@
 
     private static final Settings unboundedWriteQueue = Settings.builder().put("thread_pool.write.queue_size", -1).build();
 
+    public static final Settings settings = Settings.builder()
+        .put(ShardIndexingPressureSettings.SHARD_INDEXING_PRESSURE_ENABLED.getKey(), false).build();
```

Comment:
Tests should cover both?

Reply:
These tests is for node level Indexing pressure and hence this setting should be off here. For ``true`` we it covered as part of ``ShardIndexingPressureIT.java``